### ISSUE TITLE
Mentor Roll Call! - Saturday May 5th, 2018

### ISSUE THREAD

**Issue by jarrettch** [nodeschool/los-angeles#159]
On **Saturday May 5th**, NodeSchool will be at **Codesmith** and we will be going over [`learnyounode`](https://github.com/workshopper/learnyounode). 

It will be from **2-5pm** and you can register here on [Eventbrite](https://nodeschoolla.eventbrite.com?aff=git) as a mentor 

Are you in??

### Confirmed
- [x] @jarrettch
- [ ] you?

### Maybe 
- [ ] @rouzbeh84
- [ ] @seanwestfall
- [ ] @ashwell
- [ ] @d48
- [ ] @jcblw 
- [ ] @gr2m
- [ ] you?

**Comment by gr2m** [comment]
I’ll try to make it, I put it in my calendar

**Comment by jarrettch** [comment]
No, that's for the monthly meetup. Unfortunately we may not be able to do
NodeSchool at Codesmith this upcoming month. I'm trying to secure a new
location now so if you know anyone that can accommodate us that would be
awesome!

**Comment by 10Dimensional** [comment]
Hey all! Gregor spoke to me about you all needing a new location. We'd be down to host at @ [replicated](https://www.replicated.com/) on the 5th. Can you guys give me an estimated head count and some things you would need? We have some extra tables and conference rooms available. Thanks! :)

**Comment by jarrettch** [comment]
OMG, thank you so much @10Dimensional Yeah we usually have at most 15 people, and just need tables/wifi. We usually take care of the food/drinks ourselves at NodeSchool. We also your address and any other info needed to locate the office/parking. We can coordinate that here or you can email me at jarrett.harris@js.la

Also if you or another person at replicated wants to give a quick intro about your company to the people there that's totally cool too!

**Comment by 10Dimensional** [comment]
No problem! Happy to help out 😄 

Here's our location, as well as where people can park: https://www.google.com/maps/d/u/0/viewer?mid=1wnrRrkqg5owA3384pGVJZUOel8s&ll=34.02984805143769%2C-118.37218535&z=17

Address is 5913 Blackwelder St, Culver City, CA 90232

My email address is sloane@replicated.com or heyimsloane@gmail.com--I'm cool with coordinating either way.

And yeah! That'd be great! I can also see if one anyone wants to mentor/help out too--there's a few people at Replicated that are familiar with Node
